nomad/job_endpoint.go

```diff
@@ -433,6 +438,53 @@ func (j *Job) Register(args *structs.JobRegisterRequest, reply *structs.JobRegis
 	return nil
 }
 
+// specChanged checks to see if the job spec has changed. If the job is multiregion,
+// it checks all regions. This handles the case of a multiregion job that has been
+// stopped in one or more regions being restarted via the run command.
+func (j *Job) specChanged(existingJob *structs.Job, args *structs.JobRegisterRequest) (bool, error) {
```

Comment:
This method covers much more behavior than its internal call to `Job.SpecChanged`, and all of the behavior changes are specific to multiregion deployments:
* The method name is misleading.
* This should probably be a stub call in OSS with all the real logic only in ENT (which is the only repo it can be tested in anyways).

Reply:
Good point. On it.